apps/src/redux/instructions.js

```diff
@@ -0,0 +1,76 @@
+var _ = require('../lodash');
```

Comment:
File-level comment would be nice, especially for team members that haven't had much exposure to redux.